src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreRT.cs

```diff
@@ -151,11 +156,26 @@ public ManagedObjectWrapperHolder(ManagedObjectWrapper* wrapper)
             }
         }
 
-        internal unsafe struct IUnknownVftbl
+        internal unsafe class NativeObjectWrapper
         {
-            public delegate* unmanaged<IntPtr, ref Guid, out IntPtr, int> QueryInterface;
-            public delegate* unmanaged<IntPtr, uint> AddRef;
-            public delegate* unmanaged<IntPtr, uint> Release;
+            private IntPtr _externalComObject;
+            private ComWrappers _comWrappers;
+            public GCHandle ProxyHandle;
```

Comment:
I would keep it as `_proxyHandle`. (Some of the places in the file are actually still refering to the field as `_proxyHandle`.)

Reply:
Done.